librad/src/git.rs

```diff
@@ -286,12 +287,15 @@ pub mod project {
             paths: &Paths,
             sources: &git2::Repository,
         ) -> Result<ProjectId, Error> {
-            let mut meta = meta::Project::new(&self.name, &PeerId::from(self.key.clone()));
-            meta.default_branch = self.default_branch;
-            meta.description = self.description;
-            meta.rel = self.rel;
-
-            GitProject::init(paths, &self.key, sources, meta, self.founder.clone())
+            // FIXME[ENTITY]: add certifier instead of peer
```

Comment:
I think this will resolve itself with #106: the repo init will just take an `Entity`.